airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/embedder.py

```diff
@@ -72,7 +72,7 @@ def check(self) -> Optional[str]:
             return format_exception(e)
         return None
 
-    def embed_chunks(self, chunks: List[Chunk]) -> List[List[float]]:
+    def embed_chunks(self, chunks: List[Chunk]) -> List[Optional[List[float]]]:
```

Comment:
These typing changes are necessary because I touched the file - our current mypy check is only considering changes files, this is how these violations could get in.